microsoft/vs-servicehub
# Merge latest Library.Template

- Bump dotnet-coverage from 17.10.1 to 17.10.2 (#250)
- Remove compliance tasks
- Bring back PoliCheck
- Drop Codeql.Enabled variable
- File bugs for APIScan issues
- Move Policheck to the 1ES PT
- Disable Policheck on non-compliance runs
- Fix missing TeamEmail and TeamName in insertVS task
- Drop ComponentGovernance from the pipeline
- Recommend prettier
- Identify AzP-related YAML files
- Add missing $ characters in pipeline
- Bump Microsoft.NET.Test.Sdk from 17.8.0 to 17.9.0 (#251)
- Publish artifacts even on failed pipelines
- Switch job disabling style
- Remove `EnableSBOMSigning` variable
- Share variables across all entrypoints
- Fix Expand-Template.ps1
- Clarify purpose of nuGetServiceConnection
- Drop Ref12Analyze task
- Fix publish for scheduled pipeline runs
- Add missing elements for turnkey localization
- Fix LocBin to not fail before a build occurs
- Drop `outputParentDirectory` from release pipelines
- Fix release pipeline
- Bump .NET SDK from 8.0.100 to 8.0.201
- Bump xunit.runner.visualstudio from 2.5.6 to 2.5.7 (#254)
- Bump dotnet-coverage from 17.10.2 to 17.10.3 (#252)
- Bump xunit from 2.6.6 to 2.7.0 (#253)
- Add team as reviewer for insertion PRs
- Couple APIScan and TSAUpload under the same switch
- Add OptProf support to Library.Template
- Disable NOTICE file generation for OptProf runs
- Update ubuntu image names
- Publish logs on build/test failures for mac/linux
- OptProf pipeline fixes
- Fix `notice@0` task on non-default branches
-  Disable the generation of SBOMs for artifacts in unsigned builds
- Fix `ShouldSkipOptimize` parameter references
- Make mac/linux agents trivially excludable
- Clarify code comment
- Fix assigning reviewers to insertion PRs
